ehitchhiking-ui/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "semi": false,
```

Comment:
To my mind this option should be true (https://prettier.io/docs/en/options.html#semicolons)

Reply:
done